src/back/services/events.ts

```diff
@@ -152,4 +159,58 @@ export class EventsService {
 
     return profiles
   }
+
+  static async commented(eventId: string | undefined, event: string | undefined, discoursePost: DiscourseWebhookPost) {
+    try {
+      if (
+        !eventId ||
+        !event ||
+        event !== 'post_created' || //TODO: we don't register editions or deletions
+        (await EventModel.alreadyRegistered(eventId)) ||
```

Comment:
Maybe rename to `isCommentAlreadyRegistered`?

Reply:
How about `isDiscourseEventRegistered`?  Cause I'm not certain we are not going to register other discord events (like topic creations, or editions/deletions), and they all share this event id